### ISSUE TITLE
Rotation/Tilt of markers

### ISSUE THREAD

**Issue by hurnjoo** [tangrams/tangram-es#1002]
Hello, this is a question about marker.
I'm trying to add a marker in order to indicate current my position and direction.

Adding an icon to map is successful, but I can't change tilt/rotation of markers.
Is there a way to change tilt/rotation of markers?

This is my sample code for adding marker.
```cpp
    {
        std::string markerStyling = "{ style: 'icons', sprite: 'plane', size: [25px, 25px], order: 100, collide: false }";
        if (!marker) {
            marker = map->markerAdd();
            map->markerSetStyling(marker, markerStyling.c_str());
        }
        map->markerSetPointEased(marker, p, 0.2, EaseType::linear);
    }
```

**Comment by blair1618** [comment]
Once https://github.com/tangrams/tangram-es/pull/925 is reviewed and merged you will be able to use the `flat` and `angle` styling properties for point markers.

**Comment by blair1618** [comment]
Resolved by #925
